cron/cron.txt

```diff
@@ -0,0 +1,13 @@
+######################
+# CantusDB Cron Jobs #
+######################
+
+# Note: This is set up to run on the production server. If you want to run it on your local machine, you will need to change the paths.
+
+# min   hour    day     month   weekday command
+0       4       *       *       *       bash /home/ubuntu/code/CantusDB/cron/postgres/daily.sh
+10      4       *       *       0       bash /home/ubuntu/code/CantusDB/cron/postgres/weekly.sh
+20      4       1       *       *       bash /home/ubuntu/code/CantusDB/cron/postgres/monthly.sh
+30      4       1       1       *       bash /home/ubuntu/code/CantusDB/cron/postgres/yearly.sh
+40      4       1       *       *       bash /home/ubuntu/code/CantusDB/cron/management/manage.sh populate_is_last_chant_in_feast; bash /home/ubuntu/code/CantusDB/cron/management/manage.sh populate_next_chant_fields
```

Comment:
This appears to be back to calculating `is_last_chant_in_feast` before calculating `next_chant`? `is_last_chant_in_feast` depends on `next_chant`.